src/tracks/playabletrack/wavetrack/ui/WaveClipAdjustBorderHandle.cpp

```diff
@@ -386,85 +359,86 @@ UIHandlePtr WaveClipAdjustBorderHandle::HitAnywhere(
    const AudacityProject* pProject,
    const TrackPanelMouseState& state)
 {
-    const auto rect = state.rect;
+   assert(waveTrack->IsLeader());
 
-    auto px = state.state.m_x;
+   const auto rect = state.rect;
 
-    auto& zoomInfo = ViewInfo::Get(*pProject);
+   const auto px = state.state.m_x;
 
-    std::shared_ptr<WaveClip> leftClip;
-    std::shared_ptr<WaveClip> rightClip;
+   auto& zoomInfo = ViewInfo::Get(*pProject);
 
-    //Test left and right boundaries of each clip
-    //to determine which kind of adjustment is
-    //more appropriate
-    for (auto& clip : waveTrack->GetClips())
-    {
-        if (!WaveChannelView::ClipDetailsVisible(*clip, zoomInfo, rect))
-           continue;
-
-        auto clipRect = ClipParameters::GetClipRect(*clip.get(), zoomInfo, rect);
+   std::shared_ptr<WaveTrack::Interval> leftInterval;
+   std::shared_ptr<WaveTrack::Interval> rightInterval;
 
-        //double the hit testing area in case if clip are close to each other
```

Comment:
Maybe the comment should be kept to explain `* 2`

Reply:
I think it's a leftovers I forgot to remove when we disabled adjacent clip border adjustment, added a new commit that simplifies git testing